app/resources/hebrew_view.json

```diff
@@ -0,0 +1,1462 @@
+[
```

Comment:
I'm afraid we need to think of a better way, that will hold for many years. It's not practical to ask the user to update the Hebrew dates every year :(
Maybe we can use [`pylauch`](https://github.com/simlist/pyluach) for that?

Reply:
i change for this and also for parshot feature
what do you think?